framework/PageFramework.tsx

```diff
@@ -5,20 +5,20 @@ import { PageAlertToasterProvider } from './PageAlertToaster';
 import { PageDialogProvider } from './PageDialogs/PageDialog';
 import { PageNavSideBarProvider } from './PageNavigation/PageNavSidebar';
 import { PageNotificationsProvider } from './PageNotifications/PageNotificationsProvider';
-import { SettingsProvider } from './Settings';
-import { FrameworkTranslationsProvider } from './useFrameworkTranslations';
+import { PageSettingsProvider } from './PageSettings/PageSettingsProvider';
 import { PageBreadcrumbsProvider } from './PageTabs/PageBreadcrumbs';
+import { FrameworkTranslationsProvider } from './useFrameworkTranslations';
 
 /**
  * The `PageFramework` component bundles up all the context providers in the Ansible UI framework in a convienent component for framework consumers. Examples of internal context providers are translations, navigation, settings, alerts, and dialogs.
  *
  * @example
  * <PageFramework>...</PageFramework>
  */
-export function PageFramework(props: { children: ReactNode }) {
+export function PageFramework(props: { children: ReactNode; defaultRefreshInterval: number }) {
```

Comment:
Worth adding a default/common value for this one then we do not to override this one in several places?